src/main/cs/Infinispan/HotRod/RemoteCacheManager.cs

```diff
@@ -30,7 +30,23 @@ public class RemoteCacheManager
     {
         private Infinispan.HotRod.SWIG.RemoteCacheManager manager;
         private IMarshaller marshaller;
+        private IMarshaller argMarshaller;
         private Configuration configuration;
+        /// <summary>
+        /// Marshaller for the remote execution arguments. Defaulted to JBasicMarshaller class.
+        /// </summary>
+        public IMarshaller ArgMarshaller
```

Comment:
This setter/getter seems redundant. We should re-use the common marshaller that can be set via configuration.Marshaller(). Setting two different marshallers seems too complicated.